build.gradle

```diff
@@ -261,7 +261,7 @@ subprojects { subproject ->
         testCompile "io.rest-assured:rest-assured:${restAssuredVersion}"
         testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
-        testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.23.0-ACA-112_PR164'
+        testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.22.0'
```

Comment:
Not 5.23.0 to match the Data Store PR?

Reply:
Will eventually use a `befta-fw` pre-release containing some callback URLs pointing to a specific PR URL - this will be based on version 5.23.0.

This change was just to get a PR build running on pipeline